workflows/templates/ncn-reboot-common.yaml

```
@@ -0,0 +1,305 @@
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: ncn-reboot-common
+  namespace: argo
+spec:
+  templates:
+    - name: verify-pre-reboot-checks
+      inputs:
+        parameters:
+          - name: dryRun
+          - name: media_host
+            value: 'ncn-m001'
+      dag:
+        tasks:
+          - name: verify-pre-reboot-checks
+            templateRef:
+              name: ssh-template
+              template: shell-script
+            arguments:
+              parameters:
+                - name: dryRun
+                  value: "{{inputs.parameters.dryRun}}"
+                - name: mediaHost
+                  value: "{{inputs.parameters.media_host}}"                         
+                - name: scriptContent
+                  value: |
+                    echo "INFO NCN pre-reboot checks"
+
+                    echo "INFO Starting platform health check"
+                    result=$(/opt/cray/platform-utils/ncnHealthChecks.sh 2>&1)
+                    if [[ "$result" == *"FAILED"* ]]; then
+                      echo "ERROR platform health check has failed"
+                      exit 1
+                    else
+                      echo "INFO Successfully completed platform health check"
+                    fi
+                    
+                    echo "INFO Checking the status of the slurmctld pods"
+                    result=$(kubectl get  pod -n user -lapp=slurmctld -o jsonpath="{.items[*].status.phase}" 2>&1)
+                    if [[ "$result" != "Running" ]]; then
+                      echo "ERROR $(kubectl get  pod -n user -lapp=slurmctld -o jsonpath="{.items[*].metadata.name}") is not running"
+                      exit 1
+                    else
+                      echo "INFO $(kubectl get  pod -n user -lapp=slurmctld -o jsonpath="{.items[*].metadata.name}") is running"
+                    fi
+                    
+                    echo "INFO Checking the status of the slurmdbd pods"
+                    result=$(kubectl get  pod -n user -lapp=slurmdbd -o jsonpath="{.items[*].status.phase}" 2>&1)
+                    if [[ "$result" != "Running" ]]; then
+                      echo "ERROR $(kubectl get  pod -n user -lapp=slurmdbd -o jsonpath="{.items[*].metadata.name}") is not running"
+                      exit 1
+                    else
+                      echo "INFO $(kubectl get  pod -n user -lapp=slurmdbd -o jsonpath="{.items[*].metadata.name}") is running"
+                    fi
+
+                    echo "INFO Validating BGP peering sessions"
+                    result=$(GOSS_BASE=/opt/cray/tests/install/ncn goss \
+                      -g /opt/cray/tests/install/ncn/tests/goss-switch-bgp-neighbor-aruba-or-mellanox.yaml \
+                      --vars=/opt/cray/tests/install/ncn/vars/variables-ncn.yaml validate 2>&1)
+                    if [[ $? -ne 0 ]]; then
```

I don't see the `result` being used, only the exit value is used, so you can just send all the output to /dev/null
```suggestion
                    if ! GOSS_BASE=/opt/cray/tests/install/ncn goss \
                      -g /opt/cray/tests/install/ncn/tests/goss-switch-bgp-neighbor-aruba-or-mellanox.yaml \
                      --vars=/opt/cray/tests/install/ncn/vars/variables-ncn.yaml validate >/dev/null 2>&1
                    then
```